### DISCUSSION TITLE
[feature request] Change EarTrumpet' icon depending on input

### DISCUSSION THREAD

**Discussion by angelopampalone** [File-New-Project/EarTrumpet#695] [Ideas]
Hi, would be cool to have EarTrumpet change icon depending on the audio playback device that is selected.

My  use case is:
Pc with Headphones and monitor output. I switch back and fort between these 2 output and would be handy to have EarTrumpet's icon change to one (say a headphone icon) when the output selected is the headphone and to a different one say the classic EarTrumpet's icon when the selected output is the other one.

**Comment by vertigo220** [comment]
If nothing else, the default icon should be slightly different from the Windows sound settings icon to make it clear which is which. I only just found you can change it to use the legacy icon, but I prefer the default one, since it's more recognizable, I just wish I could more easily tell between the two.
